src/main/java/edu/umich/ctools/sectionsUtilityTool/Utils.java

```diff
@@ -249,4 +281,46 @@ public static HttpResponse executeApiCall(HttpUriRequest clientRequest) {
 		return response;
 	}
 
+	public static boolean didApiReturnedWithOutErrors(String errMsg, ApiResultWrapper arw) {
+		if (arw.getStatus() != HttpStatus.SC_OK) {
+			M_log.error(errMsg);
+			return false;
+		}
+		return true;
+	}
+
+	public static String getLTICustomParam(HttpServletRequest request, String ltiParam){
+		HashMap<String, Object> customValuesMap = getLTICustomParams(request);
+		for (String LtiParamKey : customValuesMap.keySet()) {
+			if (LtiParamKey == ltiParam) {
+				return (String) customValuesMap.get(ltiParam);
+			}
+		}
+		return null;
+	}
+
+	public static String generateSisCourseId() {
+		Random rand = new Random();
+		int ranNum = rand.nextInt(40) + 1;
```

Comment:
40 should at least be a constant.  A constant with an explanation is a lot better.